scalecube-configuration/pom.xml

```diff
@@ -11,12 +13,11 @@
   <packaging>jar</packaging>
 
   <name>scalecube-configuration</name>
-  <url>http://maven.apache.org</url>
 
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <serviceName>${project.artifactId}</serviceName>
     <mainClass>io.scalecube.server.ConfigurationServiceRunner</mainClass>
+    <extratags>${env.TRAVIS_COMMIT}</extratags>
```

Comment:
the same as https://github.com/scalecube/scalecube-organization-service/pull/58#discussion_r244578246